### PR TITLE
updated the comments api request

### PR DESCRIPTION
**Pull request by lizapincsak**
## Description
Juan and I added a put request to edit the case comment from 'Null' to whatever the admin types in. We added a put request in caseRouter and updateComment in the caseModel. We then updated the README.md documentation for this route.

Fixes # (issue)
Allows FE to update comment status from Null to a new comment. 

## Type of change
Added a new BE endpoint for the FE to access.

Please delete options that are not relevant.
- [x] New feature (non-breaking change which adds functionality)
- [x] This change requires a documentation update

## Checklist:
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have removed unnecessary comments/console logs from my code
- [x] I have made corresponding changes to the documentation if necessary (optional)
- [x] My changes generate no new warnings
- [x] I have checked my code and corrected any misspellings
- [x] No duplicate code left within changed files
- [x] Size of pull request kept to a minimum
- [x] Pull request description clearly describes changes made & motivations for said changes

### CODE REVIEW

File: api/cases/caseModel.js [hunk 1]
```diff
@@ -86,6 +86,11 @@ const casesByState = () => {
     .groupBy('case_origin_state');
 };
 
+//comments
+const updateComment = (id, updatedComment) => {
+  return db('cases').where({ case_id: id }).update(updatedComment);
+};
+
```
Comment (Poonchy) [inline api/cases/caseModel.js:93]: Might not be necessary. Test locally with the update function.

**Review by Poonchy — CHANGES_REQUESTED** [review]
Test with the update instead of updateComment function. If it works, I recommend removing updateComment. Other than that, routing and readme looks good.

**Review by juancaruizc — APPROVED** [review]
Awesome job Liza!

**Review by Poonchy — APPROVED** [review]
Looks good to me now. Clean and simple 👍🏻

**Pull request merged**